bedrock/firefox/templates/firefox/welcome/page8.html

```diff
@@ -0,0 +1,100 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+{% from "macros-protocol.html" import hero with context %}
+
+{% add_lang_files "firefox/welcome/page8" %}
+
+{% extends "firefox/welcome/base.html" %}
+
+{# L10n: HTML page title #}
+{% block page_title %}{{ _('Firefox protects your privacy automatically') }}{% endblock %}
+
+{% block body_class %}{{ super() }} welcome-page8{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-firefox-welcome-8' %}
+{% set _utm_campaign = 'welcome-8-protection' %}
+
+
+{% block content_intro %}
+{% call hero(
+    title=_('Your privacy respected, <strong>automatically</strong>'),
+    desc=_('Firefox automatically protects your privacy, so you’re free to live your life every time you get online.'),
+    class='mzp-t-firefox mzp-t-dark',
+    include_cta=True,
+    heading_level=1
+  ) %}
+
+  <p class="primary-cta protection-report">
+    <a href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?src=external-www.mozilla.org-welcom-7&utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}" class="mzp-c-button mzp-t-product" id="button" rel="external">{{ _('View your protection report') }}</a>
+  </p>
+
+  {% endcall %}
+{% endblock %}
+
+{% block content_primary %}
+  <div class="body-primary">
+    <div class="c-picto-block t-adjacent-image">
+      <div class="c-picto-block-image">
+        <img class="c-emphasis-box-logo" src="{{ static('img/icons/tracking-protection.svg') }}" width="48px" alt="">
+      </div>
+
+      <h3 class="c-picto-block-title">{{ _('Enhanced Tracking Protection') }}</h3>
+      <div class="c-picto-block-body">
+        <p>{{ _('Automatically block sites from following you around the internet.') }}</p>
+        <a class="protection-report" href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?src=external-www.mozilla.org-welcom-7&utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}" data-cta-text="See what’s blocked" data-cta-type="button"><strong>{{ _('See what’s blocked') }}</strong></a>
```

Comment:
Should this link be opening the protection report as well? If so, you might need to use `document.querySelectorAll('.protection-report')` in the JS?